src/app/ui/timeline/timeline.cpp

```diff
@@ -1958,7 +1958,7 @@ void Timeline::onAddCel(DocEvent& ev)
 
 void Timeline::onAfterRemoveCel(DocEvent& ev)
 {
-  invalidateLayer(ev.layer());
+  ui::execute_now_or_enqueue([this, &ev] { invalidateLayer(ev.layer()); });
```

Comment:
Make a copy of the DocEvent parameter because it's a local variable and might not exist in the UI thread later:
```suggestion
  ui::execute_now_or_enqueue([this, ev] { invalidateLayer(ev.layer()); });
```

Reply:
You are right! Thanks!